actions.go

```diff
@@ -51,7 +52,7 @@ type CustomActionsExternalActionsConnection struct {
 type CustomActionsTriggerDefinitionsConnection struct {
 	Nodes      []CustomActionsTriggerDefinition
 	PageInfo   PageInfo
-	TotalCount int
+	TotalCount int `graphql:"-"`
```

Comment:
This was found when digging through code - its a pattern we've switched to so just cleaning it up here